packages/components/src/components/Input/index.tsx

```diff
@@ -25,11 +26,9 @@ export const InputComponent = styled.input(
   })
 );
 
-const Label = styled.label(
+const Label = styled(Text)(
   css({
-    fontSize: 2,
-    paddingBottom: 2,
-    color: 'sidebar.foreground',
+    marginBottom: 2,
```

Comment:
The responsibility of laying out the elements falls on the container (based on the use case) . So components should not have any margin on them inside.

as each of our component is extended from Element - you can now use - <Label marginBottom={2}>

Reply:
Used the first one in the other comment